### ISSUE TITLE
Vertical line bottom of the text field

### ISSUE THREAD

**Issue by rajeshdewle** [LiewJunTung/pin_code_text_field#13]
if we added theme data for focus border in input decoration theme that time automatically added vertical line bottom of the text field.
```
ThemeData(
  inputDecorationTheme: InputDecorationTheme(
    focusedBorder: OutlineInputBorder(
        borderSide: BorderSide(
        color: Color(0xFF9FA0A5),
        ),
      ),
   )
)
```

**Comment by LiewJunTung** [comment]
Fixed in 1.3.5

**Comment by rajeshdewle** [comment]
The same problem with this code, I hope you can solve it
```
ThemeData(
  inputDecorationTheme: InputDecorationTheme(
    enabledBorder: OutlineInputBorder(
      borderSide: BorderSide(
        color: Color(0xFF9FA0A5),
        ),
      ),
    ),
  )
)
```
